backends/postgres/postgres_backend_test.go

```diff
@@ -497,3 +497,92 @@ results_loop:
 		t.Error(err)
 	}
 }
+
+func TestJobEnqueuedSeparately(t *testing.T) {
+	connString, conn := prepareAndCleanupDB(t)
+	const queue = "SyncThing"
+	maxRetries := 5
+	done := make(chan bool)
+	defer close(done)
+
+	timeoutTimer := time.After(30 * time.Second)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	enqueuer, err := neoq.New(ctx,
+		neoq.WithBackend(postgres.Backend),
+		postgres.WithConnectionString(connString),
+		postgres.WithSynchronousCommit(false),
+		neoq.WithLogLevel(logging.LogLevelDebug),
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer enqueuer.Shutdown(ctx)
+
+	consumer, err := neoq.New(ctx,
+		neoq.WithBackend(postgres.Backend),
+		postgres.WithConnectionString(connString),
+		postgres.WithSynchronousCommit(false),
+		neoq.WithLogLevel(logging.LogLevelDebug),
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer consumer.Shutdown(ctx)
+	h := handler.New(queue, func(_ context.Context) (err error) {
+		done <- true
+		return
+	})
+
+	err = consumer.Start(ctx, h)
+	if err != nil {
+		t.Error(err)
+	}
+
+	// Wait a bit more before enqueueing
+	time.Sleep(10 * time.Second)
```

Comment:
Let's get rid of this. I was waiting for `Start` to start up for a small number of milliseconds in some tests, but it doesn't seem to be necessary, because I've not done that in other tests and they don't seem flaky.